Include/internal/pycore_import.h

```diff
@@ -5,6 +5,23 @@
 extern "C" {
 #endif
 
+
+struct _import_runtime_state {
```

Comment:
Also move the global import lock to runtime state?

Reply:
Yeah, that's next. :)